src/v/syschecks/syschecks.cc

```diff
@@ -53,6 +53,7 @@ ss::future<> disk(const ss::sstring& path) {
     return ss::check_direct_io_support(path).then([path] {
         return ss::file_system_at(path).then([path](auto fs) {
             checklog.info0("Detected file system type is {}", to_string(fs));
+            if (fs == ss::fs_type::ext2) {
                 checklog.warn(
                   "Path: `{}' is on ext4, not XFS. This will probably work, "
```

Comment:
nit: perhaps update to "ext4-like filesystem"

Reply:
@andrwng - does this comment apply to line 56: `if (fs == ss::fs_type::ext2) {`?

Note that `ss::fs_type` is part of seastar.